lamindb/core/_data.py

```diff
@@ -248,6 +249,7 @@ def describe(self: Artifact | Collection, print_types: bool = False):
 
     # Print entire message
```

Comment:
```suggestion
```
self explanatory code here

Reply:
Yes, agreed. Just don't want to mess with this here because Sunny works on `describe()` in the other PR and the more changes the more conflicts.